src/content/docs/apis/rest-api-v2/basic-functions/extract-metric-timeslice-data.mdx

```diff
@@ -253,7 +253,7 @@ Sometimes the output data's granularity may be too fine, or the time period for
 
 ```
 curl -X GET 'https://api.newrelic.com/v2/applications/$APPID/metrics/data.xml' \
```

Comment:
@nwhite222 this instance of $APPID doesn't seem to follow the pattern of $APP_ID (with the underscore). I think this could be anything we want, but could you check with our subject matter expert to confirm that we'd want to be consistent here?